dist/src/test/java/io/camunda/zeebe/shared/management/BackupEndpointStandaloneTest.java

```diff
@@ -31,14 +31,14 @@ public void shouldCallTakeWhenIsStandalone() {
 
   @Test
   public void shouldCallListWhenIsStandalone() {
-    backupEndpointStandalone.list();
-    verify(backupEndpoint).list();
+    backupEndpointStandalone.query(null);
+    verify(backupEndpoint).query(null);
   }
 
   @Test
   public void shouldCallGetWhenIsStandalone() {
-    backupEndpointStandalone.status(11L);
-    verify(backupEndpoint).status(11L);
+    backupEndpointStandalone.query("11");
+    verify(backupEndpoint).query("11");
```

Comment:
🙃 No given-when-then blocks 😞. Consistency is key.

https://github.com/camunda/camunda/blob/main/docs/testing.md#general-conventions